src/libstd/io/util.rs

```diff
@@ -48,9 +64,24 @@ pub fn copy<R: Read, W: Write>(reader: &mut R, writer: &mut W) -> io::Result<u64
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct Empty { _priv: () }
 
-/// Creates an instance of an empty reader.
+/// Constructs a new handle to an empty reader.
 ///
 /// All reads from the returned reader will return `Ok(0)`.
+///
+/// # Examples
+///
+/// Reading a bunch of zeroes, for fun:
```

Comment:
This doesn't actually read any zeros, but instead doesn't read anything (e.g. doesn't modify the buffer passed in)